api/openai_client.py

```
@@ -521,8 +525,15 @@ async def acall(
     def from_dict(cls: type[T], data: Dict[str, Any]) -> T:
         obj = super().from_dict(data)
         # recreate the existing clients
-        obj.sync_client = obj.init_sync_client()
-        obj.async_client = obj.init_async_client()
+        try:
+            obj.sync_client = obj.init_sync_client()
+            # Initialize async client lazily to avoid env var issues during deserialization
+            obj.async_client = None
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)